src/test/java/com/google/cloud/spanner/jdbc/it/ITJdbcScriptTest.java

```diff
@@ -217,24 +217,12 @@ public void test03_TestGetReadTimestamp() throws Exception {
 
   @Test
   public void test04_TestGetCommitTimestamp() throws Exception {
-    // Skipping test as Commit Timestamp not supported in POSTGRES
-    assumeFalse(dialect.dialect == Dialect.POSTGRESQL);
     try (CloudSpannerJdbcConnection connection = createConnection(env, database)) {
```

Comment:
Do we need a try block?

Reply:
Yes, JDBC connections are auto-closeables, and should therefore be enclosed in try-with-resources blocks whenever possible. This ensures that the connection is closed at the end of the block, regardless what happens (e.g. exceptions, early returns, just reaching the end of the block, etc.).

Failing to close JDBC connection can cause resource leaks.